src/win/udp.c

```diff
@@ -28,6 +28,17 @@
 #include "stream-inl.h"
 #include "req-inl.h"
 
+#if defined(MCAST_JOIN_SOURCE_GROUP) && defined(MCAST_LEAVE_SOURCE_GROUP)
```

Comment:
Would it be possible to consolidate with the same code in `src/unix/udp.c`?

Reply:
Yes, It's possible. Can I place this defines in `uv.h` header?

Reply:
moved to uv.h